Block/Product.php

```diff
@@ -19,6 +18,8 @@
 use Magento\Framework\Pricing\PriceCurrencyInterface;
 use Magento\Store\Model\ScopeInterface;
 use Magento\Catalog\Model\ProductFactory;
+use Magento\Framework\Module\Manager as ModuleManager;
+use Yotpo\Yotpo\Helper\RichSnippets as YotpoRichSnippets;
```

Comment:
Check that Yotpo is still part of core in 2.3.5

Reply:
https://docs.magento.com/m2/ce/user_guide/marketing/yotpo-reviews-intro.html
https://support.yotpo.com/en/article/magento-2-installing-yotpo